src/h2load.cc

```diff
@@ -252,6 +255,61 @@ void rate_period_timeout_w_cb(struct ev_loop *loop, ev_timer *w, int revents) {
 } // namespace
 
 namespace {
```

Comment:
Yeah, it can but I just followed their own convention in the file. For every new function in the anonymous namespace, they have written in a new `namespace { }`.